React/game/src/redux/auth/actions.js

```diff
@@ -1,17 +1,51 @@
 import authService from '@services/authService';
 
+import loadService from '../../services/loadServices';
+
 export const actionsTypes = {
-  AUTH_LOGIN: 'AUTH_LOGIN'
+  AUTH_LOGIN: 'AUTH_LOGIN',
+  AUTH_LOGIN_ERROR: 'AUTH_LOGIN_ERROR',
+  LOADING_APP: 'LOADING_APP'
 };
 
 const actionCreators = {
   onLogin: (email, password) => async dispatch => {
     const response = await authService.login(email, password);
-    console.log(response);
-    dispatch({
-      type: actionsTypes.AUTH_LOGIN,
-      payload: true
-    });
+    const token = response.data.id;
+    const userId = response.data.userId;
+    if (response.ok) {
+      loadService(email, password);
```

Comment:
`loadService` is a very ambiguos name. Load what? try to be clearer when naming files/functions/variables